README.md

```diff
@@ -1,2 +1,81 @@
-# Movie-List
-Search your favorite movies and tv shows on Movie Database
+# Movie List
+> Search for info about your favorite movies and tv shows
+
+### Built With
+
+- Major languages: HTML, CSS, JavaScript
+- Frameworks: N/A
+- Technologies used: Webpack, API
+
+### You can view it live [here](https://github.com/Estherstarr/Movie-List)
+
+### Watch our project presentation [here](#)
+
```

Comment:
- [ ] In the README, please put the correct link so that README looks professional.

Reply:
The following changes were made
Implement the comment section to update correctly
Update ReadMe